utils/_context/_scenarios.py

```diff
@@ -113,6 +113,15 @@ def session_start(self):
             self.close_targets()
             raise
 
+        from utils._context.containers import _get_client
+
+        try:
+            weblog_container = _get_client().containers.get("system-tests-weblog")
```

Comment:
in the class `EndToEndScenario`, this is accessible threw `self.weblog_container`

Reply:
I moved to EndToEndScenario class.